generated/nifake/nifake.proto

```diff
@@ -29,9 +31,10 @@ service NiFake {
   rpc GetANumber(GetANumberRequest) returns (GetANumberResponse);
   rpc GetAStringOfFixedMaximumSize(GetAStringOfFixedMaximumSizeRequest) returns (GetAStringOfFixedMaximumSizeResponse);
   rpc GetAnIviDanceString(GetAnIviDanceStringRequest) returns (GetAnIviDanceStringResponse);
-  rpc GetAnIviDanceWithATwistString(GetAnIviDanceWithATwistStringRequest) returns (GetAnIviDanceWithATwistStringResponse);
```

Comment:
This looks like a bad merge. You should not remove tests for `ivi-dance-with-a-twist` with strings.

Reply:
The metadata was replaced with `GetAnIviDanceWithATwistArray` API that returns array instead of string. This was done as per Ryan's comment [here](https://github.com/doshirohan/grpc-device/pull/84/#discussion_r636525048) on the PR.

Reply:
The PR comment is a bit ambiguous, but I would interpret his comment to mean remove `GetPatternPinIndexes` from nifake and introduce a new `GetAnIviDanceWithATwistArray` _in addition to_ `GetAnIviDanceWithATwistString`.

Reply:
Created tech debt https://dev.azure.com/ni/DevCentral/_workitems/edit/1499185 for tracking this.